DEVELOPMENT.md

```diff
@@ -0,0 +1,143 @@
+# Development
+
+Please make sure to read the [Contributing guideline](./CONTRIBUTING.md) first.
+
+## How to manage the BigAnimal Bearer Token?
+
+As described in the [Readme](./README.md#getting-an-api-token) file, you can use the `get-token.sh` script to fetch a Bearer Token.
+
+Another common practice is using the [Biganimal CLI](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/).
+
+### Using BA CLI to help initializing Provider credentials
+
+1. [Install the BA CLI](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/#installing-the-cli) and [jq - Command Line JSON Processor ](https://stedolan.github.io/jq/).
+1. [Authenticate as a valid user and create a credential](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/#installing-the-cli). This command will direct you to your browser.
+```shell
+biganimal create-credential \
+  --name "ba-user1" \
+  --address "portal.biganimal.com" \
+  --port "443"
```

Comment:
The default values for address and port are "portal.biganimal.com" and "443", respectively. Hence, it's not mandatory to include them in the command "biganimal create-credential --name "ba-user"".

Reply:
This is the default example we give in [the EDB Docs](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/#authenticate-as-a-valid-user) as well, that's why I wanted to keep it aligned with our official docs. 

But true, they are optional. I'm in favor of keeping them, if you don't have a strong objection.

Reply:
Sure, I agree with you